sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -597,11 +597,13 @@ case class PartitionStatistics(numFiles: Int, totalSize: Long)
  * The syntax of this command is:
  * {{{
  *   ALTER TABLE table RECOVER PARTITIONS;
- *   MSCK REPAIR TABLE table;
+ *   MSCK REPAIR TABLE table [{ADD|DROP|SYNC} PARTITIONS];
  * }}}
  */
 case class AlterTableRecoverPartitionsCommand(
```

Comment:
@cloud-fan @dongjoon-hyun Should we rename this to `RepairTableCommand` (or `MsckRepairTableCommad`) since `ALTER TABLE .. RECOVER PARTITIONS` is just a specific case of `MSCK REPAIR TABLE table [{ADD|DROP|SYNC} PARTITIONS]`, and it doesn't support `{DROP|SYNC} PARTITIONS`?

Reply:
Well, it means many additional changes including the doc changes at line 594, `Recover Partitions in ALTER TABLE`. Why don't we do that renaming in another PR separately if you want?

Reply:
Here is the PR https://github.com/apache/spark/pull/31635 for renaming.